src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -110,6 +111,7 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
         .withPoolingOptions(poolingOptions)
         .withReconnectionPolicy(new ExponentialReconnectionPolicy(1000, reconnectMaxDelay.toMillis))
         .withQueryOptions(new QueryOptions().setFetchSize(fetchSize))
+        .withPort(port)
```

Comment:
I was under the impression that the previous behavior was very much intended, you _either_ configure one host+port or a list of contact points and in the second case the host+port is ignored.